R-package/DESCRIPTION

```diff
@@ -0,0 +1,38 @@
+Package: tfio
```

Comment:
Wondering if for cosmetic reasons is it possible to change it to tensorflow-io to match python package?

Reply:
R package name cannot contain dashes. Also I was trying to be consistent with other existing R interfaces to TF components:
* `tf.estimator`: https://github.com/rstudio/tfestimators
* `tf.dataset`: https://github.com/rstudio/tfdatasets